data/tlg1763/tlg001/tlg1763.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,423 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader xml:lang="eng">
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="grc">Γρηγορίου τοῦ Κορινθίου περὶ τρόπων</title>
+                <author>Tryphon Grammaticus</author>
+                <editor>Leonhard von Spengel</editor>
+                <funder>Center for Hellenic Studies</funder>
+                <respStmt>
+                    <resp>Published original versions of the electronic texts</resp>
+                    <orgName ref="https://www.opengreekandlatin.org">Open Greek and Latin</orgName>
+                    <persName role="principal">Gregory Crane</persName>
+                    <persName role="principal">Leonard Muellner</persName>
+                    <persName role="principal">Bruce Robertson</persName>
+                </respStmt>
+
+                <respStmt>
+                    <resp>Proofreading and CTS conversion</resp>
+                    <persName>Michael Konieczny</persName>
+                    <orgName/>
+                </respStmt>
+            </titleStmt>
+
+            <publicationStmt>
+                <publisher>Open Greek and Latin</publisher>
+                <idno type="filename">tlg1763.tlg001.1st1K-grc1.xml</idno>
+                <availability>
+                    <licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a
+                        Creative Commons Attribution-ShareAlike 4.0 International License</licence>
+                </availability>
+                <date>2020-07-05</date>
```

Comment:
I've been making this the GitHub date (as opposed to the Lace export date) but it doesn't matter much to me